source/hash_table.c

```diff
@@ -531,9 +595,10 @@ int aws_hash_table_remove(struct aws_hash_table *map,
     return AWS_OP_SUCCESS;
 }
 
-int aws_hash_table_foreach(struct aws_hash_table *map,
-    int (*callback)(void *baton, struct aws_hash_element *pElement), void *baton)
-{
+int aws_hash_table_foreach(
+    struct aws_hash_table *map,
+    int (*callback)(void *context, struct aws_hash_element *pElement),
+    void *context) {
     struct hash_table_state *state = map->pImpl;
```

Comment:
it would be cool if there was a rule that enforced whitespace after a multiline blocking opening like this, so the args don't blend in with the function body